pandas/tests/tools/test_to_datetime.py

```diff
@@ -858,22 +858,31 @@ def test_to_datetime_dt64s_and_str(self, arg, format):
         "dt", [np.datetime64("1000-01-01"), np.datetime64("5000-01-02")]
     )
     def test_to_datetime_dt64s_out_of_bounds(self, cache, dt):
```

Comment:
should the name of the test change, now that they're no longer out-of-bounds?

Reply:
gentle reminder

Reply:
@jbrockmendel last one, sorry 😄

Reply:
already renamed to reflect it is only out of ns bounds

Reply:
I missed that, thanks!